partial-flashing.js

```diff
@@ -699,7 +699,6 @@ let PartialFlashing = {
 
             let timeout = new Promise((resolve, reject) => {
                 setTimeout(() => {
-                    PartialFlashingUtils.log("Resetting micro:bit timed out");
```

Comment:
Yep, the timeout still completes but the result of the promise is ignored

> Should we store the ID to the timeout reachable by the other parts of the promise so that this setTimeout can be cancelled when the operation is successful?

Sound like the neatest way of doing this